insights/tests/client/init/test_write_pidfile.py

```diff
@@ -5,14 +5,17 @@
 
 @patch("insights.client.write_to_disk")
 @patch("insights.client.os.getpid")
-def test_write_pidfile(getpid, write_to_disk):
+@patch("insights.client.utilities.get_parent_process")
```

Comment:
This patch is not correct. You need the path in the namespace from which _get_parent_process_ is used, not where it is defined.

```suggestion
@patch("insights.client.get_parent_process")
```